Makefile.custom.mk

```diff
@@ -2,7 +2,7 @@
 generate:
 	./hack/generate-kustomize-patches.sh
 	@rm -rf helm/cluster-api-provider-azure/templates/*.yaml
-	@cp helm/cluster-api-provider-azure/files/copy/* helm/cluster-api-provider-azure/templates/
+	@cp config/helm/copy/* helm/cluster-api-provider-azure/templates/
```

Comment:
In the current version, crd-install job creates configmaps for all files under helm/copy folder. See https://github.com/giantswarm/cluster-api-provider-azure-app/blob/main/helm/cluster-api-provider-azure/templates/crd-install/crd-configmap.yaml#L6

It is harmless since we don't refer them in the kustomization file  https://github.com/giantswarm/cluster-api-provider-azure-app/blob/main/helm/cluster-api-provider-azure/files/kustomization.yaml

I noticed it since it broke my PR because of long CM name. Then I moved the folder.

Reply:
@giantswarm/team-phoenix I just checked `cluster-api-provider-aws-app` and the problem exists there too.